implementations/rust/ockam/ockam_api/src/nodes/service.rs

```diff
@@ -356,15 +361,16 @@ impl NodeManager {
                 Some(f) => BootstrapedIdentityStore::new(Arc::new(f), repository.clone()),
             });
 
+        info!("create the secure channels service");
```

Comment:
Regarding the log level, I wrote a small document some months ago while reviewing the ockam_api and ockam_command crates. I also added some general guidelines used in popular repos.

The main problem I found with defaulting to `info` is that logs tend to be very verbose.